dynalab_cli/init.py

```diff
@@ -211,13 +213,31 @@ def initialize_field(self, key, value):
             raise NotImplementedError(f"{key} not supported in setup_config")
 
     def initialize_task(self, key, value):
-        tasks = get_tasks()
-        while value not in tasks:
+        tasks, task_codes = get_tasks()
```

Comment:
Do we need the `tasks` variable?

Reply:
We do need the tasks variable here as we get the task i/o definition from it. But thanks for pointing this out, I will replace `tasks` with `_` in other places where we don't use it